provision/ubuntu/kernel-next.sh

```diff
@@ -37,4 +37,8 @@ cd ..
 sudo dpkg -i linux-*.deb
 sudo ln -sf /boot/System.map-$(uname -r) /boot/System.map
 
+cd tools/perf
+sudo make
```

Comment:
Do we really need to run this with privileges?

Reply:
Unfortunately, yes:
```
vagrant@k8s2:~/k/tools/perf$ make
  BUILD:   Doing 'make -j2' parallel build
  HOSTCC   fixdep.o
fixdep.c:170:1: fatal error: opening dependency file ./.fixdep.o.d: Permission denied
 }
 ^
compilation terminated.
/bin/sh: 1: cannot create ./.fixdep.o.cmd: Permission denied
/home/vagrant/k/tools/build/Makefile.build:96: recipe for target 'fixdep.o' failed
make[3]: *** [fixdep.o] Error 2
Makefile:43: recipe for target 'fixdep-in.o' failed
make[2]: *** [fixdep-in.o] Error 2
/home/vagrant/k/tools/build/Makefile.include:5: recipe for target 'fixdep' failed
make[1]: *** [fixdep] Error 2
Makefile:69: recipe for target 'all' failed
make: *** [all] Error 2
```

Let me try dropping `sudo` from `make` above.

Reply:
Uh, weird, I didn't need it when I tried this locally (on the existing image).